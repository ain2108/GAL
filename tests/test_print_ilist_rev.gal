int main()
{
	
	ilist x;
    ilist rev_x;

	x = [11];
	x = iadd(10,x);
	x = iadd(9,x);
	x = iadd(8,x);
	x = iadd(7,x);
	x = iadd(1,x);
	x = iadd(2,x);
	x = iadd(3,x);

	x = irev(x);

	print_ilist(x);
	return 1;
}