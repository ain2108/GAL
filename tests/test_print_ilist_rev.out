->11::10::9::8::7::1::2::3::